doc/languages-frameworks/javascript.section.md

```diff
@@ -444,6 +444,7 @@ stdenv.mkDerivation (finalAttrs: {
   pnpmDeps = pnpm.fetchDeps {
     inherit (finalAttrs) pname version src;
     hash = "...";
+    fetcherVersion = 2;
```

Comment:
I don't like that we write the fetcherVersion behind the hash. For all other fetchers we always we have the hash last which I think is usually a good indication that any change before it influences the hash.

Reply:
I'm not sure if changing it after the merge is worth it, it seems to be a minor thing to me and would cause merge conflicts for (probably) tens of PRs